aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,140 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarDarkModeExample,
+  AvatarDataTableExample,
+  AvatarDataTableIconExample,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+  MarketplaceNavigationalCards,
+} from '../../examples';
+
+## When to use
+
+Avatars are used to enable users to quickly identify an object, whether that be a specific user, account, or other entity.
+
+Avatars are always paired with additional information, such as a user's name, for better clarity. If possible, use an image or logo over initials.
+
+The consistent dimensions of an avatar ensure that readability and scanability of content surrounding the avatar is not compromised. For example, when scanning a [data table column that contains an avatar of a company’s logo](/avatar#datatable) and the company name, the company name stays aligned from one row to the next. 
```

Comment:
Same with this link, I get a page not found error

Reply:
fixed